packages/atomic/src/components/atomic-result-list/atomic-result-list.tsx

```diff
@@ -85,25 +85,24 @@ export class AtomicResultList implements InitializableComponent {
     this.host
       .querySelectorAll('atomic-result-template')
       .forEach(async (resultTemplateElement) => {
-        const conditions = await resultTemplateElement.getConditions();
-        const fields = await resultTemplateElement.getFields();
-        this.resultTemplatesManager.registerTemplates({
-          content: resultTemplateElement.innerHTML,
-          conditions,
-          fields,
-          priority: 1,
-        });
+        const template = await resultTemplateElement.getTemplate();
+        if (!template) {
+          this.templateHasError = true;
+          return;
+        }
+        this.resultTemplatesManager.registerTemplates(template);
       });
   }
 
   private get results() {
     return this.resultListState.results.map((result) => (
       <atomic-result
-        key={result.uniqueId}
+        key={result.raw.permanentid}
         part="list-element"
         result={result}
         engine={this.bindings.engine}
-        innerHTML={Mustache.render(
+        // TODO: decide to get rid of Mustache or not
+        content={Mustache.render(
```

Comment:
What is the downside of leaving Mustache in?

Reply:
it might just not be really necessary

Reply:
With Mustache, this means that people will be able to "easily" render things inside their result templates, right ?

Like, if they just wanted to display a field value or something.

Otherwise, they would always need to create an atomic component to be able to display anything in templates ?

🤔

Reply:
>  if they just wanted to display a field value or something

For that we have the atomic-result-value